plugins/action_update_review_status/src/updateReviewsStatuses.test.ts

```diff
@@ -258,6 +256,90 @@ describe('Update review_responses for second review submission by reviewer2 when
               reviewStatus: ReviewStatus.Pending,
             },
           ],
+          updatedReviewAssignments: [],
+        },
+      })
+    })
+  })
+})
+
+describe('Update review_response to submit review with LOQ to applicant by consolidator2', () => {
+  // Setup database
+  beforeAll(async (done) => {
+    await DBConnect.query({
+      text: `
+      UPDATE public.review_response SET is_visible_to_applicant = true WHERE id = 5004;
+      UPDATE public.review_response SET status = 'SUBMITTED' WHERE id = 5020;
+      UPDATE public.review_response SET status = 'SUBMITTED' WHERE id = 5021;
+      UPDATE public.review_response SET status = 'SUBMITTED' WHERE id = 5022;
+      UPDATE public.review_response SET status = 'SUBMITTED' WHERE id = 5023;
+      UPDATE public.review_response SET status = 'SUBMITTED' WHERE id = 5024;
+      INSERT INTO public.review_decision (id, decision, review_id)
+        VALUES (DEFAULT, 'LIST_OF_QUESTIONS', 12);
+      INSERT INTO public.review_status_history (id, review_id, status)
+        VALUES (DEFAULT, 12, 'SUBMITTED');
+      `,
+      values: [],
+    })
+    done()
+  })
+
+  test('Consolidation submitted to applicant with List of questions => Lock other lower review assignments', () => {
+    return updateReviewsStatuses({
+      parameters: {
+        triggeredBy: 'REVIEW',
+        changedResponses: [
+          {
+            applicationResponseId: 4150,
+            templateElementId: 4001,
+            reviewResponseDecision: 'APPROVED',
+          },
+          {
+            applicationResponseId: 4151,
+            templateElementId: 4002,
+            reviewResponseDecision: 'APPROVED',
+          },
+          {
+            applicationResponseId: 4152,
+            templateElementId: 4003,
+            reviewResponseDecision: 'APPROVED',
+          },
+          {
+            applicationResponseId: 4153,
+            templateElementId: 4004,
+            reviewResponseDecision: 'APPROVED',
+          },
+          {
+            applicationResponseId: 4154,
+            templateElementId: 4005,
+            reviewResponseDecision: 'DECLINED',
+          },
+        ],
+      },
+      // @ts-ignore
```

Comment:
Maybe a quick comment why we're ignoring?
`// @ts-ignore -- ignore missing properties on applicationData`

Reply:
Done: f1c5e28